packages/server/modules/core/graph/plugins/logging.ts

```diff
@@ -107,37 +107,44 @@ export const loggingPluginFactory: (deps: {
           apollo_query_duration_ms: Date.now() - apolloRequestStart
         })
 
-        for (const err of ctx.errors) {
```

Comment:
this was the reason why there were many "req failed" entries logged for a single request